### ISSUE TITLE
[Feature] Add site to blocked list before timer expires

### ISSUE THREAD

**Issue by hajile00** [matchu/Strict-Workflow#64]
I find myself getting distracted by sites I didn't know I was distracted by, such as xkcd.com, and I can't edit the blocked list until the timer expires.

**Comment by truth1ness** [comment]
Agreed. It makes sense that you aren't allowed to remove sites but you should totally be able to add sites if you need to restrict yourself more during the timer.

**Comment by tarmacx** [comment]
I have a fork which already have that.
It's in a context menu if you right click.
https://github.com/tarmacx/Strict-Workflow
Currently looking for some testers before releasing it.
